hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerReader.java

```diff
@@ -261,6 +268,30 @@ public void verifyAndFixupContainerData(ContainerData containerData)
     }
   }
 
+  private boolean isMatchedLastLoadedECContainer(
```

Comment:
Can we move this logic to containerSet class instead of container reader. There is a function containerSet.addContainer(container, boolean overwrite) we could use that if we want to write to disk.